tests/integration/components/text-field-test.js

```diff
@@ -107,8 +106,12 @@ test('On update calls format function', function(assert) {
 
   run(() => {
     $input.val('2');
-    $input.trigger('blur');
   });
 
-  assert.equal(this.format($input.val()), '20');
+  assert.equal($input.val(), '2');
```

Comment:
Add a message to this test, so we know what is been tested. (Like "Does not format on changing")